docs: recap blog for goose nyc meetup (#2264)
block/goose: documentation/blog/2025-04-17-goose-goes-to-NY/index.mdx

---
title: "Codename Goose Goes to New York"

> description: "Goose lands in NYC for its second ever community meetup"

Reviewer: ```suggestion
description: "Goose lands in NYC for its second community meetup"
```

authors: 
    - ebony
---

import ImageCarousel from '@site/src/components/ImageCarousel';
import swag from './swag2.jpg';
import swag1 from './swag1.jpg';
import swag2 from './swag.jpg';
import swag3 from './swag3.jpg';
import focus from './focus.jpg';
import focus2 from './focus2.jpg';
import focus3 from './focus3.jpg';
import focus4 from './focus5.jpg';
import focus5 from './focus4.jpg';
import speaker from './speaker.jpg';
import speaker1 from './speaker1.jpg';
import fun from './fun.jpg';
import fun1 from './fun1.jpg';

![blog cover](cover.png)

# Codename Goose Goes to New York 🗽

> We brought Goose to New York and NYC was one for the books.

Reviewer 2: ```suggestion
We brought Goose to New York City, and the event was one for the books.
```

feel free to switch it up. I'm just thinking the New York then NYC seems repetitive.

> Over 100 people registered, plus 87 on the waitlist, and we had a packed room full of folks who were curious, thoughtful, and ready to dive in. Some were developers already exploring Goose and MCP, others were totally new to the world of AI agents. That’s the beauty of Goose it’s for developers *and* non-developers.

Reviewer: ```suggestion
Over 100 people registered, plus 87 on the waitlist, and we had a packed room full of folks who were curious, thoughtful, and ready to dive in. Some were developers already exploring Goose and MCP, others were totally new to the world of AI agents. That’s the beauty of Goose, it’s for developers *and* non-developers.
```

> The energy was there from the moment the event began, music playing, people grabbing pizza, relaxing, and genuinely networking. We had lightning talks, a Goose-themed game, hands-on hacking, and yeah… a few Ebbs IPAs might’ve ended up in people’s backpacks by the end of the night.

Reviewer: ```suggestion
The energy was there from the moment the event began - music, pizza, and authentic networking. We had lightning talks, a Goose-themed game, hands-on hacking, and yeah… a few Ebbs IPAs might’ve ended up in people’s backpacks by the end of the night.
```

> — *Nitya Narasimhan, AI @ Microsoft*

> “The live demos, open discussions, and practical exercises made it an exceptional learning experience. Feeling energized and inspired to explore how Goose and MCP can be leveraged in future data engineering projects!”  
> — *Manikumarreddy Gajjela, Full Stack Developer*

> “Everyone seems bullish on Model Context Protocol (MCP) these days… Goose is an open-source, on-machine AI agent built to automate your tasks. The coolest part? You can even plug in your own hosted Ollama LLM with Goose!”  
> — *Rijul Dahiya, NYU Courant*

> “Great event – great mix of introductory and technical material as well as fun, and I liked that it provided a bunch of time at the beginning and end that was unstructured.”  
> — *Matthew Hill*

## Moments We Loved ❤️

- 🍕 Pizza, music, and people catching up before things even kicked off  
- 🎮 The Goose game- yes, it got competitive and oddly dark lol
- 🧳 Folks traveled from Philly and DC just to attend  

> - 📸 Having a photographer meant we could actually *enjoy* the night  

Reviewer 2: maybe like having a photographer to capture candid, fun moments cuz they dont have context that we were snapping pics by ourselves in the Boston event.

Author: great catch thanks

- 🧠 A bunch of thoughtful “I’m not a dev but…” questions that we *loved* hearing  

<ImageCarousel id="test-carousel" images={[speaker1,speaker, fun, fun1, focus2]} />

At the end, people were asking when the next one is and yes, we may have let it slip early…

## Up Next: Goose Flies South 🛫🐣

Our next Goose Meetup is already locked in!

👉 **Atlanta – Wednesday, April 30**  
🕕 **6:00 PM – 8:30 PM ET**  
📍 [RSVP here](https://lu.ma/x9ccqruq)

> ## Things We’re Thinking About for Next Time

Reviewer: recommend removing this section

Every event teaches us something. Here’s what we’re carrying forward:

> - **Let everyone off the waitlist 2 days before.** RSVP drop-offs are real, and this keeps the energy high.

Reviewer 2: wait i feel like we should leave this part out and keep that as info for our team only. feel free to let me know why you're including this.

Reviewer 2: okay i like the what we learned..but i dont wanna tell them about the waitlist thing yet LOLLL.

- **Feature a community showcase.** Would love to highlight projects from folks using Goose and give the mic to others next time.

## Big Thanks 🙌

Huge shoutout to **[Tania Chakraborty](https://www.linkedin.com/in/taniachakraborty/)** and **[Anthony Giuliano](https://www.linkedin.com/in/anthonygiuliano1/)** for being there, and to everyone who came through, asked questions, helped out, and made it a night to remember.

The community truly made this one special, people stayed after to chat, helped clean up without being asked, and brought such a good energy into the room. We couldn’t have asked for a better crowd.

We’ll see you in Atlanta 👀

<head>
  <meta property="og:title" content="Codename Goose Goes to New York" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="https://block.github.io/goose/blog/2025/04/14/community-atruelight4" />

>   <meta property="og:description" content="Goose lands in NYC for its second ever community meetup" />

Reviewer: ```suggestion
  <meta property="og:description" content="Goose lands in NYC for its second community meetup" />
```
